src/integration/pages/onDemandTVPage/pashto/__snapshots__/amp.test.js.snap

```diff
@@ -160,24 +160,6 @@ exports[`AMP On Demand T V Page Header Navigation link should match text and url
 
 exports[`AMP On Demand T V Page Header Navigation link should match text and url 9`] = `"کالم - /pashto/columns"`;
 
-exports[`AMP On Demand T V Page Header Navigation link should match text and url 1`] = `"کور پاڼه - /pashto"`;
```

Comment:
I don't understand why these have been reverted - they should be generated by #6282

Reply:
They have been duplicated